scripts/init.sh

```diff
@@ -126,6 +135,8 @@ start-frequency-interval)
     --rpc-external \
     --rpc-cors all \
     --rpc-methods=Unsafe \
+    --sealing-create-empty-blocks \
```

Comment:
Probably not a big deal given the limited scope/use of this script, but worth noting that this is a significant change in behavior for `start-frequency-interval` which will now keep forming empty blocks every `${interval}` seconds. For anyone that tends to leave their dev chain running, this means that older blocks will roll off the history unless you manually tweak the params to enable archiving.

Again, not a big deal as it's a dev script, but worth noting.

Reply:
I think it's more closer to reality to have this as the default for `start-frequency-interval`, but as you said might not work as some previously expected so we can maybe parking lot it to get more opinions.

Reply:
removing this so that it does not violate the current expectations